bigflow/bigquery/job.py

```diff
@@ -14,12 +16,14 @@ def __init__(self,
                  id=None,
                  retry_count=DEFAULT_RETRY_COUNT,
                  retry_pause_sec=DEFAULT_RETRY_PAUSE_SEC,
+                 job_execution_timeout=DEFAULT_JOB_EXECUTION_TIMEOUT,
```

Comment:
I think `execution_timeout` is a better name, the `job_` prefix is redundant (because you set the execution_timeout on a job) and not consistent with the rest of the parameters (there is, for example, `retry_count` param, not `job_retry_count`.

Reply:
I've added job_ prefix because this timeout is passed to airflow job

Reply:
removed